client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -3,155 +3,134 @@
  */
 import { __ as NO__ } from '@wordpress/i18n';
 import { Button } from '@wordpress/components';
-import { isEmpty, map } from 'lodash';
-import { useDispatch, useSelect } from '@wordpress/data';
-import React, { useCallback, useState } from 'react';
+import { isEmpty } from 'lodash';
+import { useSelect } from '@wordpress/data';
+import React, { useState } from 'react';
 import classNames from 'classnames';
 /**
  * Internal dependencies
  */
-import { SiteType, isFilledFormValue } from '../store/types';
+import { isFilledFormValue } from '../store/types';
 import { STORE_KEY } from '../store';
 import VerticalSelect from './vertical-select';
+import SiteTypeSelect, { siteTypeOptions } from './site-type-select';
+import SiteTitle from './site-title';
 import './style.scss';
 
-const siteTypeOptions: Record< SiteType, string > = {
-	[ SiteType.BLOG ]: NO__( 'with a blog' ),
-	[ SiteType.BUSINESS ]: NO__( 'for a business' ),
-	[ SiteType.PORTFOLIO ]: NO__( 'to share a portfolio' ),
-	[ SiteType.STORE ]: NO__( 'for a store' ),
-};
+interface StepProps {
+	isActive: boolean;
+	onExpand: () => void;
+	onSelect: () => void;
+}
 
-export default function OnboardingEdit() {
-	const [ lastSelectedQuestion, setLastSelectedQuestion ] = useState( null );
+const Step1 = ( { isActive, onExpand, onSelect }: StepProps ) => {
```

Comment:
Nitpick, let's give them descriptive names -- the order might be subject to change, but the functionality will likely remain the same

```suggestion
const SiteTypeStep = ( { isActive, onExpand, onSelect }: StepProps ) => {
```

Reply:
Thanks for the suggestion!
They have their own names and files now.